en/asgardeo/docs/guides/api-security/aws/securing-aws-http-apis.md

```diff
@@ -0,0 +1,279 @@
+---
+template: templates/quick-start.html
+heading: Secure a Pharmacy User API with AWS API Gateway and Asgardeo
```

Comment:
Shall we keep the heading generic?

```suggestion
heading: Securing APIs with AWS API Gateway and Asgardeo
```

Reply:
Addressed in [e4fc8aa](https://github.com/wso2/docs-is/pull/5253/commits/e4fc8aa8a7f3140b9f3447455ff995b2ad2ef2ac)